speedtest/server.go

```diff
@@ -42,6 +46,29 @@ type Server struct {
 	doer *http.Client
 }
 
+// CustomServer Given a URL string, return a new Server object, with as much
+// filled in as we can
+func CustomServer(s string) (*Server, error) {
+	if !strings.HasSuffix(s, "/upload.php") {
+		return nil, errors.New("please use the full URL of the server, ending in '/upload.php'")
```

Comment:
💅 
```suggestion
		return nil, errors.New("Please use the full URL of the server, ending in '/upload.php'")
```

Reply:
Emmm...I think there is no problem here, the go standard specification of error string is start with lowercase.

Reply:
Fixed by https://github.com/showwin/speedtest-go/pull/93